gector/gec_predictor.py

```diff
@@ -88,7 +94,9 @@ def predict_batch(self, sentences: List[str]) -> List[JsonDict]:
                 - corrected_words
             For an explanation of each of these see `Seq2Labels.decode()`.
         """
-        return self.predict_batch_json([{"sentence": sentence} for sentence in sentences])
+        return self.predict_batch_json(
+            [{"sentence": sentence} for sentence in sentences]
+        )
 
     @overrides
     def predict_instance(self, instance: Instance) -> JsonDict:
```

Comment:
Predict instance should probably create a batch from a single instance and call the revised predict_batch_instance method you've written. Currently, I think batch prediction and single instance prediction (what would most likely be used in the engine) would give you differing results.